.gitignore

```diff
@@ -1 +1,2 @@
-target/
\ No newline at end of file
+target/
+local_adl/
```

Comment:
Nice catch.  With the latest [changes](https://github.com/mongodb/mongo-odbc-driver/blob/24237421dd0d32de2c2b289a1cc6e8be2d695a1a/resources/run_adf.sh#L44), this is now `local_adf/`

Reply:
Oh right, good catch. I'll update this right now!